fluster/fluster.py

```diff
@@ -55,10 +55,10 @@ def __init__(
         reference: bool = False,
         summary: bool = False,
         keep_files: bool = False,
-        threshold: int = None,
-        time_threshold: int = None,
+        threshold: Union[int, None] = None,
```

Comment:
You can use `Optional[int]` as a shorthand for `Union[int, None]`.

Reply:
Totally agree. I did a fixup: https://github.com/fluendo/fluster/pull/81/commits/d77322c8990c9e8ddd0bf70b96b92d7eb4a536ae

Squash + merge?